### PR TITLE
Contributors

### PR DESCRIPTION
**Pull request by undertherain**

### CODE REVIEW

**Comment by undertherain** [comment]
please squash when merging

File: README.rst [hunk 1]
```diff
@@ -53,9 +53,9 @@ Contributors
 The original version was developed in 2017 by `Aleksandr Drozd <https://blackbird.pw/>`_.
 Since then to the project contributed (in alphabetical order)
 
-- Kevin Brown
+- `Kevin Brown <https://kevinabrown.github.io/>`_
 - Mateusz Bysiek
-- Aleksandr Drozd
+- `Aleksandr Drozd <https://blackbird.pw/>`_
 - Artur Podobas
```
Comment (vatai) [inline README.rst:59]: Doesn't Arthur also have a homepage. He was showing it to you in Fukuoka!

Comment (undertherain) [reply README.rst:59]: could not find it... lemmy ask him directly

**Comment by undertherain** [comment]
added his famous iframe site)

**Pull request merged**